### ISSUE TITLE
Brought back the ability to use checkbox column for multiple selection

### ISSUE THREAD

**Issue by kshashov** [vaadin-component-factory/selection-grid-flow#19]
Hey there!
As far as I understand, now it is possible to select rows by clicking with the left mouse button while holding the Ctrl key (or Meta key) pressed. Is there any chance of getting the selection checkbox column back?
![selection_column](https://user-images.githubusercontent.com/3830719/122069450-42c0f680-ce06-11eb-8875-225c7454515a.png)

If you don't have time to implement this, we would be very grateful if you could give a little hint on what needs to be done for this and if there are any pitfalls.

**Comment by jcgueriaud1** [comment]
The component does not have a API for that you can try to revert it this way: (Just after setSelectionMode)
```
 grid.getElement().executeJs("if (this.querySelector('vaadin-grid-flow-selection-column')) { this.querySelector('vaadin-grid-flow-selection-column').hidden = false }"); 
```

I have no idea if the component is working well after the changes
Can you try it and check if it's working?

**Comment by kshashov** [comment]
Yes, we've tried that already. The problem is that clicking on the checkbox will work in the same way as clicking anywhere else on the row, and we will have to use ctrl or shift for multiple selection.
![selection_grid_checkbox_column](https://user-images.githubusercontent.com/3830719/124759694-fbc7ac00-df40-11eb-805e-f0fd4e2ea7fb.gif)

**Comment by jcgueriaud1** [comment]
In my opinion, the selection with CTRL/SHIFT click shouldn't be used with the checkboxes. It's two different "modes".

In your case, the click on the checkbox is propagated. I'm not sure if you can easily know that the click is coming from the vaadin-checkbox of the selection column.

**Comment by stefanuebe** [comment]
Regarding showing the checkbox again: if https://github.com/vaadin-component-factory/selection-grid-flow/pull/22 is merged, you may override the method hideMultiSelectColumn() in a subclass, that should easily disable the hiding.
